Sources/OneWay/Effect.swift

```diff
@@ -205,4 +205,14 @@ public enum Effects {
             }
         }
     }
+    
+    /// An effect that can be generated from AsyncStream manually.
+    public struct Stream<Element>: Effect where Element: Sendable {
+        public var values: AsyncStream<Element>
+        init(
+            _ stream: AsyncStream<Element>
+        ) {
+            self.values = stream
+        }
+    }
```

Comment:
To support cases where `Swift Concurrency` is not used, I will provide an `Effect` that can be used similar to Combine's [Future](https://developer.apple.com/documentation/combine/future).
Thank you so much for the great inspiration.